### DISCUSSION TITLE
Writing to local JSON isn't working. METHOD NOT ALLOWED

### DISCUSSION THREAD

**Discussion by Earnest-Williams** [yandeu/five-server#85] [Help / Questions]
Error I'm receiving: PUT http://127.0.0.1:5500/people.json net::ERR_ABORTED 405 (Method Not Allowed)

Anyone have any thoughts on what I do to fix this?

Thanks!

**Comment by yandeu** [comment]
What are you trying to achieve? Are you making a PUT request to the five-server?

**Reply by Earnest-Williams** [reply]
`function saveToJson() {
            const json = JSON.stringify(people);
            fetch("people.json", {
                method: "put",
                body: json
            });`

**Reply by yandeu** [reply]
Unfortunately, five-server does not provide such a thing.

**Reply by Earnest-Williams** [reply]
:/ Thanks for the help!

**Reply by yandeu** [reply]
You're welcome :)
